packages/cli/config/eslint-factory.js

```diff
@@ -37,6 +47,12 @@ function createConfig(dir, extraConfig = {}) {
     env,
     parserOptions,
     ignorePatterns,
+    testFilePatterns = [
+      '**/*.test.*',
+      '**/*.stories.*',
+      'src/setupTests.*',
+      '!src/**',
```

Comment:
These changed, but don't see a mention of that in the changeset. Was it intentional?